pkg/sdk/testint/setup_test.go

```diff
@@ -120,6 +120,9 @@ func (itc *integrationTestContext) initialize() error {
 	if err != nil {
 		return err
 	}
+	if config == nil {
```

Comment:
If we enter here it means that someone has not configured the secondary profile. It may make some of the tests fail in unexpected ways (e.g. we will try to access secondaryX). I would prefer to return error here, encouraging to set up the secondary profile for tests.

Reply:
no problem.